docs/rest_api.rst

```diff
@@ -71,862 +74,584 @@ Following are the available API endpoints with which you can interact with Raide
-| 200 OK           | For a successful Query    |
-+------------------+---------------------------+
-| 404 Not Found    | If the channel does not   |
-|                  | exist                     |
-+------------------+---------------------------+
-| 500 Server Error | Internal Raiden node error|
-+------------------+---------------------------+
+      [
+          {
+              "channel_address": "0x2a65Aca4D5fC5B5C859090a6c34d164135398226",
+              "partner_address": "0x61C808D82A3Ac53231750daDc13c777b59310bD9",
+              "token_address": "0xEA674fdDe714fd979de3EdF0F56AA9716B898ec8",
+              "balance": 35000000,
+              "state": "open",
+              "settle_timeout": 100,
+              "reveal_timeout": 30
+          }
+      ]
 
+   :statuscode 200: Successful query
+   :statuscode 500: Internal Raiden node error
 
+.. http:get:: /api/(version)/tokens
 
-Querying all registered Tokens
-------------------------------
+   Returns a list of addresses of all registered tokens.
 
-By making a ``GET`` request to ``/api/<version>/tokens`` you get a list of addresses of all registered tokens.
+   **Example Request**:
 
+   .. http:example:: curl wget httpie python-requests
 
-Example Request
-^^^^^^^^^^^^^^^
+      GET /api/1/tokens HTTP/1.1
+      Host: localhost:5001
 
-``GET /api/1/tokens``
+   **Example Response**:
 
-Example Response
-^^^^^^^^^^^^^^^^
-``200 OK`` and
-
-::
+   .. sourcecode:: http
 
-    [
-        "0xea674fdde714fd979de3edf0f56aa9716b898ec8",
-        "0x61bb630d3b2e8eda0fc1d50f9f958ec02e3969f6"
-    ]
-
-Possible Responses
-^^^^^^^^^^^^^^^^^^
-
-+------------------+---------------------------+
-| HTTP Code        | Condition                 |
-+==================+===========================+
-| 200 OK           | For a successful Query    |
-+------------------+---------------------------+
-| 404 Not Found    | If the token does not     |
-|                  | exist                     |
-+------------------+---------------------------+
-| 500 Server Error | Internal Raiden node error|
-+------------------+---------------------------+
-
-
-Querying all partners for a Token
------------------------------------
-
-By making a ``GET`` request to ``/api/<version>/tokens/<token_address>/partners`` you can get a list of all partners
-you have non-settled channels with.
-
-Example Request
-^^^^^^^^^^^^^^^
-
-``GET /api/1/tokens/0x61bb630d3b2e8eda0fc1d50f9f958ec02e3969f6/partners``
-
-Example Response
-^^^^^^^^^^^^^^^^
-``200 OK``
-
-::
+      HTTP/1.1 200 OK
+      Content-Type: application/json
 
+      [
+          "0xEA674fdDe714fd979de3EdF0F56AA9716B898ec8",
+          "0x61bB630D3B2e8eda0FC1d50F9f958eC02e3969F6"
+      ]
 
-    [
-        {
-            "partner_address": "0x61c808d82a3ac53231750dadc13c777b59310bd9",
-            "channel": "/api/<version>/channels/0x2a65aca4d5fc5b5c859090a6c34d164135398226"
-        }
-    ]
+   :statuscode 200: Successful query
```

Comment:
I tried to do a check that you moved all the previous status codes correctly but the github diff makes it impossible to check. Just leaving this as a note here, that we should do a check on the status code to make sure nothing got mixed up after this PR.